Automattic/wp-calypso
# Theme Showcase: Fix alignment of the more button

#### Proposed Changes

This PR fixes the more button overlapping with the style variations in the theme card.

| Before | After |
| --- | --- |
| ![Screenshot 2023-01-19 at 10 36 35 PM](https://user-images.githubusercontent.com/797888/213470554-ce474ea3-0400-432f-9da0-591a4efbd94f.png) |  ![Screenshot 2023-01-19 at 10 36 51 PM](https://user-images.githubusercontent.com/797888/213470589-b5301ff5-8b2f-4b96-8cbf-faa4856d2285.png) |

#### Testing Instructions



* Head to the logged-in Theme Showcase. If using calypso.live, the flag `themes/showcase-i4/details-and-preview` is required.
* Ensure that the more button no longer overlaps with the style variations.
* Also test against the logged-out Theme Showcase.

#### Pre-merge Checklist
